sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/Item.kt

```diff
@@ -0,0 +1,26 @@
+package co.nimblehq.sample.compose.ui.screens.home
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import co.nimblehq.sample.compose.model.UiModel
+import co.nimblehq.sample.compose.ui.theme.Dimension
+
+@Composable
+fun Item(
+    uiModel: UiModel,
+    onClick: (UiModel) -> Unit
+) {
+    Surface(
+        modifier = Modifier
+            .fillMaxWidth()
+            .clickable(onClick = { onClick(uiModel) })
+    ) {
+        Row(modifier = Modifier.padding(Dimension.SpacingNormal)) {
+            Text(text = uiModel.id.toString())
+        }
+    }
+}
```

Comment:
@chornerman I think we can get rid of `Surface` and use just `Row` is enough. Let's create a `preview` then we can see how was it looks like. Here's some idea
```suggestion
    Row(modifier = Modifier
        .fillMaxWidth()
        .clickable(onClick = { onClick(uiModel) })
    ) {
        Text(
            modifier = Modifier.padding(Dimension.SpacingNormal),
            text = uiModel.id.toString()
        )
    }
}

@Preview(showBackground = true)
@Composable
fun ItemPreview() {
    val uiModel = UiModel(id = 1)
    Theme {
        Item(
            uiModel = uiModel,
            onClick = { uiModel.id }
        )
    }
}
```

Reply:
@Wadeewee Isn't it good practice to build composables on a `Surface` so that it has appropriate content colours? 🤔 
[Reference](https://developer.android.com/jetpack/compose/themes/material#content-color)

Reply:
@ryan-conway It seems like on the current implementation, we don't use `Surface` to handle the color anymore, but we're using it to handle `clickable ` and `padding` instead, Also, I didn't see any requirement of `color` for `background` of the `main screen` or `content` anymore. That's why I don't think we need it here 🙇‍♀️ 

And here're the result that I've got  with using `Surface` and without `Surface`
<img width="369" alt="Surface" src="https://user-images.githubusercontent.com/28002315/185533902-761bbdb8-877e-4197-b1b1-f4effc205443.png">
<img width="369" alt="NonSurface" src="https://user-images.githubusercontent.com/28002315/185533910-2cfcec88-1fd9-47b5-b917-4aecc7a1faed.png">